engine/api/authentication/ldap/ldap.go

```diff
@@ -76,14 +79,23 @@ func (d AuthDriver) CheckSigninRequest(req sdk.AuthConsumerSigninRequest) error
 
 func (d AuthDriver) GetUserInfo(ctx context.Context, req sdk.AuthConsumerSigninRequest) (sdk.AuthDriverUserInfo, error) {
 	var userInfo sdk.AuthDriverUserInfo
-	var bind = req["bind"]
+	var user = req["user"]
 	var password = req["password"]
 
-	if err := d.bind(ctx, bind, password); err != nil {
-		return userInfo, sdk.NewError(sdk.ErrUnauthorized, err)
+	if d.conf.BindDN != "" {
+		if err := d.bindDN(ctx, d.conf.BindDN, d.conf.BindPW); err != nil {
+			return userInfo, sdk.NewError(sdk.ErrUnauthorized, err)
+		}
+	} else {
+		userDN := strings.Replace(d.conf.UserDN, "{0}", ldap.EscapeFilter(user), 1)
```

Comment:
`d.conf.UserDN` doesn't seems to be initialized

Reply:
I've pushed a new commit that should fix this https://github.com/bnason/cds/commit/e1925c700f9370a78a2d4ead8772cec6ed89f911